src/Sylius/Behat/Page/Admin/ShippingMethod/CreatePage.php

```diff
@@ -126,13 +120,33 @@ protected function getDefinedElements(): array
             'channel' => '#sylius_shipping_method_channels .ui.checkbox:contains("%channel%")',
             'calculator' => '#sylius_shipping_method_calculator',
             'calculator_configuration' => '.ui.segment.configuration',
+            'weight' => '[id*="sylius_shipping_method_rules_"][id*="_configuration_weight"]',
             'code' => '#sylius_shipping_method_code',
             'name' => '#sylius_shipping_method_translations_en_US_name',
             'zone' => '#sylius_shipping_method_zone',
             'rules' => '#sylius_shipping_method_rules',
         ]);
     }
 
+    private function getValidationMessageForElement(NodeElement $element): string
+    {
+        if (null === $element) {
+            throw new ElementNotFoundException($this->getSession(), 'Field element');
+        }
```

Comment:
```suggestion
    private function getValidationMessageForElement(?NodeElement $element = null): string
    {
        if (null === $element) {
            throw new ElementNotFoundException($this->getSession(), 'Field element');
        }
```
As we made it nullable here `private function getChannelConfigurationOfLastRule(string $channelCode): ?NodeElement` as well